.bazelignore

```diff
@@ -0,0 +1 @@
+example/
```

Comment:
The example directory is its own workspace and doesn't play nice when using target globbing `//...`